SABnzbd.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python -OO
+#!/usr/bin/env python
```

Comment:
Can we leave this the same? We had many discussions about this and finally settled on this one.

Reply:
Not sure, I changed it because it would not run as-is with `/usr/bin/python`. I believe this is because the path for Python is different under snapd.